android-auto-app/src/main/java/com/mapbox/navigation/examples/androidauto/app/search/AppSearchBottomSheet.kt

```diff
@@ -1,61 +1,55 @@
 package com.mapbox.navigation.examples.androidauto.app.search
 
-import android.content.res.Configuration
 import android.os.Bundle
 import android.view.View
 import androidx.lifecycle.MutableLiveData
 import com.mapbox.search.ui.view.CommonSearchViewConfiguration
-import com.mapbox.search.ui.view.SearchBottomSheetView
-import com.mapbox.search.ui.view.category.Category
-import com.mapbox.search.ui.view.category.SearchCategoriesBottomSheetView
 import com.mapbox.search.ui.view.place.SearchPlace
 import com.mapbox.search.ui.view.place.SearchPlaceBottomSheetView
 
 class AppSearchBottomSheet(
-    private val searchBottomSheetView: SearchBottomSheetView,
     private val searchPlaceView: SearchPlaceBottomSheetView,
-    private val searchCategoriesView: SearchCategoriesBottomSheetView,
     view: View,
     savedInstanceState: Bundle?
 ) {
     val searchPlaceLiveData = MutableLiveData<SearchPlace>()
 
-    private val configuration = if (view.resources.configuration.orientation == Configuration.ORIENTATION_LANDSCAPE) {
-        SearchBottomSheetView.Configuration(
-            collapsedStateAnchor = SearchBottomSheetView.CollapsedStateAnchor.SEARCH_BAR
-        )
-    } else {
-        SearchBottomSheetView.Configuration()
-    }
+//    private val configuration = if (view.resources.configuration.orientation == Configuration.ORIENTATION_LANDSCAPE) {
```

Comment:
@narko all the code that is being commented out, is this disabling search entirely or is there an issue?

Asking because we prefer to have working code instead of commented out code. We also want to continue to support the search view.

Reply:
Oh I'm deleting this comment cause the details are all in the PR description. Sorry, I'm catching up